src/view/BeeUI.py

```diff
@@ -452,6 +454,17 @@ def showStatus(self, rank, points):
     def showWrongGuess(self, str):
         self.correctLabel.configure(text=str, font=('Arial', 25))
 
+    # Public method saveScreenshot
+    def saveScreenshot(self):
+        self.x, self.y = self.root.winfo_rootx(), self.root.winfo_rooty()
+        self.w, self.h = self.root.winfo_width(), self.root.winfo_height()
+
+        screenShot = pyautogui.screenshot(region=(self.x, self.y, self.w, self.h))
+        filepath = filedialog.asksaveasfilename(defaultextension=[(".png")], initialdir=os.getcwd())
```

Comment:
filetypes argument is needed.  make it png so that we can save the image

Reply:
fixed